test/library/standard/Reflection/cannotInspectBuiltins.prediff

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh 
+
+# filter out line numbers
+TESTNAME=$1
+OUTFILE=$2
+TMPFILE=$OUTFILE.prediff.tmp
+
+sed -E 's/^.+\.chpl:[0-9]*:/FILE.chpl:n:/' < $OUTFILE > $TMPFILE
```

Comment:
Were you getting some errors in different files than the test file itself?  Otherwise this conversion seems unnecessary

Reply:
Yes. Some of the errors come from the test file itself and some of the error comes from Reflection.chpl. This is because many of the tested functions call others in Reflection which report the error. I did try to make this a little more friendly for users by having all the errors be reported from the test file. But because many of these errors are coming from `where` clauses, changing them was breaking other tests.

Reply:
Hrm.  It might be worth opening an `error message` issue about improving that, and having a future that demonstrates the interplay cleanly.  I think you made the right call to not get bogged down in it for now, but it seems like something that would be nice to improve when we have the time